gtests/net/common/defaults.sh

```diff
@@ -54,6 +54,9 @@ sysctl -q net.ipv4.tcp_fastopen_key=a1a1a1a1-b2b2b2b2-c3c3c3c3-d4d4d4d4
 
 sysctl -q net.ipv4.tcp_syncookies=1
 
+# enable MPTCP
+sysctl -q net.mptcp.enabled=1
```

Comment:
can we do that here in this file?

Seems OK for me but we could also create a file in `gtests/net/mptcp/common/defaults.sh` setting this sysctl and then sourcing the file that is currently referenced.
Up to you :)

Reply:
```
[root@dhcp-158-243 net]# pwd
/root/packetdrill/gtests/net
[root@dhcp-158-243 net]# ls -l mptcp/common/
total 4
lrwxrwxrwx. 1 root root  24 Jun 29 16:08 defaults.sh -> ../../common/defaults.sh
-rwxr-xr-x. 1 root root 561 Jun 29 16:08 set_sysctls.py
[root@dhcp-158-243 net]# ls -l tcp/common/
total 4
lrwxrwxrwx. 1 root root  24 Jun  9 12:56 defaults.sh -> ../../common/defaults.sh
-rwxr-xr-x. 1 root root 561 Jun  9 12:56 set_sysctls.py
```
In the initial import, I found useful to do a symlink, like TCP does, to avoid different TCP settings in 2 separate folders. As long as only `net.mptcp.enable` is the only sysctl, I think we can live together with common TCP / MPTCP settings.

Reply:
Thank you for your reply!

> As long as only `net.mptcp.enable` is the only sysctl, I think we can live together with common TCP / MPTCP settings.

Fine for me! :-)